MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -194,29 +204,34 @@ public void UpdateEncoderPath(string path, string pathType)
         /// <returns><c>true</c> if the version validation succeeded; otherwise, <c>false</c>.</returns>
         private bool ValidatePath(string path, FFmpegLocation location)
         {
-            bool rc = false;
-
             if (!string.IsNullOrEmpty(path))
             {
                 if (File.Exists(path))
                 {
-                    rc = new EncoderValidator(_logger, path).ValidateVersion();
-
-                    if (!rc)
+                    var version = _encoderValidator.ValidateVersion(path);
+                    if (version != null)
                     {
-                        _logger.LogWarning("FFmpeg: {Location}: Failed version check: {Path}", location, path);
+                        if (EncoderValidator.BestVersion(version, _bestVersion))
+                        {
+                            _bestVersion = version;
+                            _bestPath = path;
+                            _bestLocation = location;
+                            return true;
+                        }
+
+                        _logger.LogInformation("FFmpeg: {Location}: version { Version } found: {Path}", location, version, path);
+                        return false;
```

Comment:
```suggestion
                        _logger.LogInformation("FFmpeg: {Location}: version {Version} found: {Path}", location, version, path);
```

don't return so following log is sent